onnxruntime/core/graph/contrib_ops/quantization_defs.cc

```diff
@@ -217,13 +217,22 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
     QuantizeBFP, 1,
     OpSchema()
         .Attr("bfp_type", "The type of BFP - must match with the BFPType enum", AttributeProto::INT)
-        .Attr("block_dims",
-              "Numbers within a bounding box will span across these dimensions."
-              "Any dimension not in this list is the same for all numbers within a bounding box."
-              "As an example, consider a 2D tensor with shape [d0, d1] and block_dims equal to [1]."
-              "Within a bounding box, all elements will be within the same row but will be from different columnns."
+        .Attr("block_dim",
+              "Each bounding box spans this dimension."
+              "Typically, the block dimension corresponds to the reduction dimension of the matrix multipication that "
+              "consumes the output of this operator."
+              "For example, for a 2D matrix multiplication A@W, QuantizeBFP(A) would use block_dim 1 and "
+              "QuantizeBFP(W) would use block_dim 2."
+              "If the bounding box size is larger than the block dimension, then each bounding box will be partially "
+              "full."
```

Comment:
Nit picking: some of the description here is unclear. I realize that a full description of the quantized-format is non-trivial. I would suggest adding a pointer to some document where things are explained in more detail. For the parts we choose to describe here, it would be great to make it clearer. For example: I assume that the real issue is about what happens when block-dimension-size is not an exact-multiple of the bounding-box-size? And that the "last" box is the one that is partially full? But does that mean it is padded or not? If it is an implementation detail, it may be better to omit the description altogether here.

Reply:
I have added a link to the paper: https://www.microsoft.com/en-us/research/publication/pushing-the-limits-of-narrow-precision-inferencing-at-cloud-scale-with-microsoft-floating-point/

You are right that if the dimension of the tensor exceeds the bounding box size, then the last box may be the one that is partially full. The nature of padding is an implementation detail. So I've omitted such details from the description.